src/main/resources/oopprj/tetrisgame/tetris/playing-view.fxml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.Pane?>
+<?import javafx.scene.control.Label?>
+
+
+
+<VBox xmlns="http://javafx.com/javafx/11.0.1"
+      xmlns:fx="http://javafx.com/fxml"
+      fx:controller="controllers.PlayingViewController"
+      style="-fx-padding: 15;"
+      stylesheets="@../../../styles/playing.css">
+
+    <HBox xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8.0.0">
+        <Button onAction="#backToMainScreen" styleClass="btn">
+            <graphic>
+                <ImageView  fx:id="homeIcon" fitHeight="30.0" fitWidth="30.0"/>
+            </graphic>
+        </Button>
+
+        <Region HBox.hgrow="ALWAYS" />
+        <ImageView  fx:id="settingIcon" fitHeight="30.0" fitWidth="30.0"/>
+    </HBox>
+
+    <HBox alignment="CENTER">
+        <ImageView  fx:id="clear30Lines" fitHeight="63.2" fitWidth="196.8"/>
+    </HBox>
+
+    <HBox spacing="15">
+        <VBox.margin>
+            <Insets top="30" />
+        </VBox.margin>
+        <HBox styleClass="box" prefWidth="123.2" prefHeight="65"  >
+            <padding>
+                <Insets top="6.0" left="6.0" right="6.0" bottom="6.0"/>
+            </padding>
+            <Label  text="Hold" style="-fx-text-fill: white; -fx-font-size: 12px;" />
```

Comment:
"style" in this place and other place  should be in 1 style of css (inline, external or internal), in our prj we use external, so you should move it to css file